src/pkg/caendr/caendr/services/sql/etl/phenotype_metadata.py

```diff
@@ -0,0 +1,70 @@
+import csv
+
+from caendr.services.logger import logger
+
+from caendr.models.datastore  import Species
+from caendr.utils.local_files import LocalDatastoreFile
+
+
+def parse_phenotype_metadata(species: Species, **files: LocalDatastoreFile):
+  """
+    Parsing function to load PhenotypeMetadata table from datastore.
+    For bulk file ouputs each trait as a new row and replicates metadata for all of them
+  """
+  
+  logger.info('Parsing function for traits metadata')
+  
+  for fname, fpath in files.items():
```

Comment:
Quick note on naming -- technically, `files.items()` will give you a list of file names (of type `str`) and file objects (of type `LocalDatastoreFile`), as opposed to just paths.  This isn't a huge deal, but it might be clearer to use e.g. `fname, f` or `file_name, file_object` or something similar.  Might make the usage of `fpath.metadata()` below a bit clearer.

Reply:
Fixed in the commit https://github.com/AndersenLab/CAENDR/commit/6dff197494274095938bbf64cef3d2accce75ea2